volumes/miovision/README.md

```diff
@@ -362,7 +362,8 @@ Adding intersections on the other hand is not as simple as removing an intersect
 
 	a) Run the [api script](https://github.com/CityofToronto/bdit_data-sources/blob/miovision_api_bugfix/volumes/miovision/api/intersection_tmc.py) with the following command line to only include intersections that we want as well as skipping the data processing process. `python3 intersection_tmc.py run-api --start_date=2020-06-15 --end_date=2020-06-16 --intersection=35 --intersection=38 --intersection=40  --pull` . `--pull` has to be included in order to skip data processing and gaps finding since we are only interested in finding invalid movements in this step. Note that multiple intersections have to be stated that way in order to be included in the list of intersections to be pulled. Recommend to test it out with a day worth of data first.
 	
-	b) First run the SELECT query below and validate those new intersection movements. The line `HAVING COUNT(DISTINCT datetime_bin::time) >= 20` is there to make sure that the movement is actually legit and not just a single observation. Then, INSERT INTO `intersection_movements` table which only has all the valid movements for old / decommissioned intersections. Keeping the decommissioned ones there too just in case that we might need those in the future.
+	b) First run the SELECT query below and validate those new intersection movements. The line `HAVING COUNT(DISTINCT datetime_bin::time) >= 20` is there to make sure that the movement is actually legit and not just a single observation. Then, INSERT INTO `intersection_movements` table which has all valid movements for intersections. These include decommissioned intersections, just in case we might need those in the future.
```

Comment:
This should maybe mention this is a starting point. As you discovered, the user should definitely validate manually

Reply:
I'm not certain whether eliminating movements performed by fewer than 20 vehicles even really makes sense, since it potentially leads to eg. u-turns allowed only along one leg. I recommend just removing that clause and asking the user to make sure all movements are physically possible, if not necessarily legal.

Reply:
Sure.